app/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html ng-app="cowsayApp">
+ <head>
+   <title>Welcome to Cowville</title>
+ </head>
+ <body>
+   <header>
+     <nav ng-controller="NavController as navCtrl">
+       <h4>cowcontrol.io</h4>
+       <ul>
+         <li ng-repeat="route in navCtrl.routes">
+           <a href="{{ route.url }}">{{ route.name }}</a>
+         </li>
+       </ul>
+     </nav>
+   </header>
+
+   <main>
+     <section
+       class="container"
+       ng-controller="CowsayController as cowsayCtrl"
+       ng-init="cowsayCtrl.current = 'squirell'">
+       <section>
+         <h2>{{ cowsayCtrl.title }}</h2> <h3> {{ cowsayCtrl.secondTitle }}</h3>
+       </section>
+
+       <div class="animal">
+       <pre class="cow">
+         {{ cowsayCtrl.update(cowsayCtrl.text) }}
+       </pre>
+     </div>
+
+
+       <form
+         ng-submit="cowsayCtrl.speak(cowsayCtrl.text)"
+         name="cowsayForm"
+         novalidate>
+
+
+      <aside class="extra">
+        <div class="extra-wrapper"> Select a cowfile
+          <select ng-model="cowsayCtrl.current">
+           <option ng-repeat="cowfile in cowsayCtrl.cowfiles" value="{{ cowfile }}">
+           </option>
```

Comment:
I found the culprit! So I guess you were missing the '{{ cowfile }}' between the option tags, so that the names could appear on the page like so:
<option ng-repeat="cowfile in cowsayCtrl.cowfiles" value="{{ cowfile }}">
    {{ cowfile }}
</option>